ArkhamOverlay.TcpUtils/AoTcpRequest.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace ArkhamOverlay.TcpUtils {
+    public enum AoTcpRequest { Unknown, GetCardInfo, ClickCardButton, ClearAll };
+
+    public static class AoTcpRequestExtensions {
```

Comment:
minor: this type of method exists natively. See: https://docs.microsoft.com/en-us/dotnet/api/system.enum.parse